src/test/store/receive-profile.test.js

```diff
@@ -38,6 +38,10 @@ import exampleSymbolTable from '../fixtures/example-symbol-table';
 import SymbolStoreDB from '../../profile-logic/symbol-store-db';
 jest.mock('../../profile-logic/symbol-store-db');
 
+// Mocking expandUrl
+import { expandUrl } from '../../utils/shorten-url';
+jest.mock('../../utils/shorten-url');
```

Comment:
I decided to mock this module because I test it more properly in its unit tests and I thought it wasn't necessary to run through it in this test file. Moreover it makes it easier to mock `fetch` calls that fetch a profile from a store.

Reply:
Thanks for the explanation, this might be nice to put as a comment in the source code.

Reply:
Ah, writing this I knew you would ask for it :-)